cylc/flow/hostuserutil.py

```diff
@@ -209,6 +209,24 @@ def is_remote_user(self, name):
                 self.remote_users[name] = True
         return self.remote_users[name]
 
+    def _is_remote_platform(self, platform):
+        """Return True if any job host in platform have different IP address
+        to the current host.
+
+        Return False if name is None.
+        Return True if host is unknown.
+
+        Todo:
+            Should this fail miserably if some hosts are remote and some are
+            not?
```

Comment:
Ok to leave this in with a TODO for now, I think there is a plan revolving around install targets?